modules/components/analytics/core/kpi/org.wso2.carbon.pc.analytics.core.kpi/src/main/java/org/wso2/carbon/pc/analytics/core/kpi/utils/DASConfigurationUtils.java

```diff
@@ -129,19 +131,19 @@ public static void setPropertyDASAnalyticsConfigured(String processName, String
                     reg.put(processAssetPath, resource);
 
                     if (log.isDebugEnabled()) {
-                        log.debug("isDasConfigedForAnalytics property in process.rxt set as true");
+                        log.debug("isDasConfigedForAnalytics property in process.rxt is set as true");
                     }
                 }
             }
         } catch (TransformerException | RegistryException e) {
-            String errMsg = "Exception in setting property isDASAnalyticsConfigured in process.rxt for the process:"
+            String errMsg = "Error in setting property isDASAnalyticsConfigured in process.rxt for the process: "
                     + processName + ":" + processVersion;
             log.error(errMsg, e);
             throw new ProcessCenterException(errMsg, e);
```

Comment:
Thanks @darshanasbg This is already fixed with 636d316f1c9c92034a9d3c7c1a8fef26dbd5ba1b